src/features/search/components/NativeCategoryValue/NativeCategoryValue.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import styled from 'styled-components/native'
+
+import { NativeCategoryIdEnumv2, SubcategoriesResponseModelv2 } from 'api/gen'
+import { getNativeCategoryValue } from 'features/search/helpers/categoriesHelpers/categoriesHelpers'
+import { Typo } from 'ui/theme'
+
+interface NativeCategoryValueProps {
+  nativeCategoryId: NativeCategoryIdEnumv2
+  data: SubcategoriesResponseModelv2 | undefined
+}
+
+export const NativeCategoryValue = ({ nativeCategoryId, data }: NativeCategoryValueProps) => {
+  const nativeCategoryValue = getNativeCategoryValue(data, nativeCategoryId)
+
+  return (nativeCategoryValue?.length ?? 0) > 0 ? (
```

Comment:
Super obscur...
```suggestion
return !!nativeCategoryValue ? (
```
ou même si tu préfères
```suggestion
return Boolean(nativeCategoryValue) ? (
```

Reply:
Puisque `nativeCategoryValue` peut être `null | undefined | string` ça te gère tous les cas, même celui du `string` vide (je ne pense pas qu'une telle chose puisse arriver mais puisque tu tiens à le vérifier avec le `.length`...)